src/test/java/org/jenkinsci/plugins/gitclient/GitClientTest.java

```diff
@@ -257,6 +257,44 @@ private String randomEmail(String name) {
         return name.replaceAll(" ", ".") + "@middle.earth";
     }
 
+    @Test
+    @Issue("JENKINS-297799")
+    /**
+     *  Changelog was formatted on word boundary prior to
+     * 72 characters with git client plugin 2.0+ when using CLI git.
+     * Was not truncated by git client plugin using JGit (And Apache version).
+     * Rely on caller to truncate first line if desired.
+     * Matching change will be included in git plugin 4.0.0
+     * to retain existing truncation behavior.
+     */
+    public void testChangelongVeryLong() throws Exception {
```

Comment:
Please correct the method name from testChangelongVeryLong to testChangelogVeryLong